Src/Plugins/AzureDevOpsConfiguration/AzureDevOpsConfiguration.csproj

```diff
@@ -11,6 +11,21 @@
     <PackageId>Sarif.PatternMatcher.AzureDevOpsConfiguration</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
+		// MSB3277 - Found conflicts with different versions of the same
+		//  dependent assembly that could not be resolved. Specifically
+		//  for Newtonsoft where we need older versions for compatibility
+		// MSB3030 - This error occurs when a file that was supposed to be
+		//  copied isn't found at the expected location.
+		//  There are copy commands that have a continueOnError=true because
```

Comment:
The text seems a little convoluted to me. How about: "Some copy commands have continueOnError=true since the copy fails, and then eventually succeeds. MSB3030 errors, in such cases, are false positives and are therefore OK to suppress. 
TODO: Investigate why the copy failure occurs even though the operation eventually succeeds."

I am not sure what value is added by saying this has been the case for almost two years. Better to add the TODO so that we can track this, and hopefully mitigate it properly.